test/libsolidity/SemanticTest.cpp

```diff
@@ -105,11 +105,12 @@ void SemanticTest::parseExpectations(istream& _stream)
 {
 	TestFileParser parser{_stream};
 	auto functionCalls = parser.parseFunctionCalls();
-	move(functionCalls.begin(), functionCalls.end(), back_inserter(m_tests));
+	std::move(functionCalls.begin(), functionCalls.end(), back_inserter(m_tests));
```

Comment:
Ah, I'm usually not sure if it's more readable to add the namespace even it's not needed. But in general I think it makes sense :)

Reply:
It is needed - there was a clash with some other function that seems to have been pulled in via an include or so.

Reply:
Ok, thanks.

Reply:
The clash most likely comes from `boost::move()`, I had that recently, too.